pkgs/tools/networking/gmrender-resurrect/default.nix

```diff
@@ -19,6 +19,14 @@ in
       sha256 = "14i5jrry6qiap5l2x2jqj7arymllajl3wgnk29ccvr8d45zp4jn1";
     };
 
+    patches = [
+      (fetchpatch {
+        url = "https://patch-diff.githubusercontent.com/raw/hzeller/gmrender-resurrect/pull/214.patch";
+        sha256 = "0fqi58viaq9jg5h5j1725qrach4c3wmfmh0q43q4r8az2pn7dszw";
+        name = "libupnp.patch";
+      })
+    ];
```

Comment:
@timokau Given that I patched the build but did not touch the version here, what should I do in regards to the package version? Does that become an unstable version nonetheless?

Reply:
No, we don't change the version for minor compatibility patches. Some distros include a patch-level in the version, but we don't need that since the full derivation name is unique anyway (with the hash).